src/ProjectSystemTools/BuildLogging/BuildLoggingToolWindow.cs

```diff
@@ -301,17 +307,27 @@ protected override int InnerQueryStatus(ref Guid commandGroupGuid, uint commandC
             var enabled = false;
             var visible = false;
             var latched = false;
-
             switch (cmd.cmdID)
             {
                 case ProjectSystemToolsPackage.StartLoggingCommandId:
                     visible = true;
-                    enabled = !_dataSource.IsLogging;
+                    enabled = !_dataSource.IsLogging();
+                    //ThreadHelper.JoinableTaskFactory.RunAsync(async () =>
```

Comment:
Is this part of what you're planning to implement?

Reply:
Yeah the commented code is going to be for when I start adding async code